internal/incident/incident.go

```diff
@@ -276,7 +265,7 @@ func (i *Incident) evaluateRules(eventID int64, causedBy types.Int) (types.Int,
 			}
 
 			i.Rules[r.ID] = struct{}{}
-			i.logger.Infof("[%s %s] rule %q matches", i.Object.DisplayName(), i.String(), r.Name)
+			i.logger.Infof("Rule %q matches on", r.Name)
```

Comment:
I wouldn't add the "on" here at the end. I think this is supposed to be a reference to the JSON that's appended to the log message, but I think the log message should be self-contained with the JSON providing additional context.